src/components/Filter/Filter.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../assets/css/colors';
+
+$color-green-dark: #265732;
+$color-gray-light: #dadada;
+
+.filterHeading {
+  background-color: $color-green-light !important;
+  color: $color-green-dark !important;
+}
+
+@media (max-width: 768px) {
+  .filterHeading:hover {
+    cursor: pointer;
+  }
+}
+
+.filterGroup {
```

Comment:
Hva med
```
.filterGroup:not(:last-child) {
```
og så flytte `className={styles.filterGroup}` fra div'en i FilterGroup til div'en som omslutter `<FilterGroup>` i LessonFilter? Da slipper vi den siste linjen under den siste filtergroupen.

Forøvrig hadde det kanskje vært bedre å rename .filterGroup til .bottomBorder eller .borderBetween eller noe slikt?